### PR TITLE
merge 4.4.3 into 4.4

### PR DESCRIPTION
**Pull request by DFolchA**
|Related issue|
|---|
|#2221|

## Description

This PR merges the latest changes in 4.4.3 into 4.4 branch.

### CODE REVIEW

File: debs/SPECS/wazuh-agent/debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+wazuh-agent (4.4.3-RELEASE) stable; urgency=low
+
+  * More info: https://documentation.wazuh.com/current/release-notes/
```
Comment (davidjiglesias) [inline debs/SPECS/wazuh-agent/debian/changelog:3]: If you use current then this gets obsolete very fast.

File: debs/SPECS/wazuh-manager/debian/changelog [hunk 2]
```diff
@@ -1,4 +1,10 @@
 
+wazuh-manager (4.4.3-RELEASE) stable; urgency=low
+
+  * More info: https://documentation.wazuh.com/current/release-notes/
```
Comment (davidjiglesias) [inline debs/SPECS/wazuh-manager/debian/changelog:4]: Same

File: stack/indexer/deb/debian/changelog [hunk 3]
```diff
@@ -1,3 +1,9 @@
+wazuh-indexer (4.4.3-RELEASE) stable; urgency=low
+
+  * More info: https://documentation.wazuh.com/current/release-notes/
```
Comment (davidjiglesias) [inline stack/indexer/deb/debian/changelog:3]: Same

**Review by davidjiglesias — APPROVED** [review]

**Review by davidjiglesias — APPROVED** [review]

**Review by davidjiglesias — APPROVED** [review]

**Review by verdx — APPROVED** [review]
LGTM

File: debs/SPECS/wazuh-agent/debian/changelog (hunk 1 above)
Comment (DFolchA) [reply debs/SPECS/wazuh-agent/debian/changelog]: Done https://github.com/wazuh/wazuh-packages/pull/2222/commits/5b9fbc36f9ff5e21d9cf38ff13eca28bc7aee0ca

File: debs/SPECS/wazuh-manager/debian/changelog (hunk 2 above)
Comment (DFolchA) [reply debs/SPECS/wazuh-manager/debian/changelog]: Done https://github.com/wazuh/wazuh-packages/pull/2222/commits/5b9fbc36f9ff5e21d9cf38ff13eca28bc7aee0ca

File: stack/indexer/deb/debian/changelog (hunk 3 above)
Comment (DFolchA) [reply stack/indexer/deb/debian/changelog]: Done https://github.com/wazuh/wazuh-packages/pull/2222/commits/5b9fbc36f9ff5e21d9cf38ff13eca28bc7aee0ca

**Review by davidjiglesias — APPROVED** [review]

**Pull request merged**